packages/template-retail-react-app/app/components/_app/index.jsx

```diff
@@ -148,8 +149,32 @@ const App = (props) => {
         history.push(path)
     }
 
+    // A major pain point for the current development of Storefront Preview is that we don't have
+    // static URLs on deployed sites. The path /mobify/bundle/123/static/storefront-preview.js
+    // changes to a different number with every deployment. Once we figure out the static URL we can
+    // update it for production, but will still need to come up with something for staging...
+
+    const storefrontPreviewClientScript =
+        process.env.NODE_ENV === 'development'
+            ? 'http://localhost:3000/mobify/bundle/development/static/storefront-preview.js'
+            : // Gotta update this manually for now... :(
+              // : 'https://runtime.commercecloud.com/storefront-preview.js'
+              null
+
+    if (!storefrontPreviewClientScript) {
+        // Deliberately aggressive because this branch exists solely for Storefront Preview development
+        return (
+            <Box className="sf-app">
+                Hey! You forgot to set a Storefront Preview client script URL!
+            </Box>
+        )
+    }
+
```

Comment:
We currently don't have a static URL for the client script for staging environments, so when you deploy you'll always have to manually specify the client script URL. That's easy to forget, so I tried to make it as in-your-face as possible.